walletnode/services/artworkregister/state/state_test.go

```diff
@@ -0,0 +1,234 @@
+package state
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const timeTestLayout = "2006-01-02 15:04:05"
+
+func newTestStates() *State {
+	timeList := createTestTimeList()
+
+	return &State{
+		statuses: []*Status{
+			{
+				CreatedAt: timeList[0],
+				Type:      StatusTaskStarted,
+				isFinal:   false,
+			}, {
+				CreatedAt: timeList[1],
+				Type:      StatusConnected,
+				isFinal:   false,
+			}, {
+				CreatedAt: timeList[2],
+				Type:      StatusTaskCompleted,
+				isFinal:   true,
+			},
+		},
+	}
+}
+
+func createTestTimeList() []time.Time {
+	time1, _ := time.Parse(timeTestLayout, "2020-01-02 07:00:00")
+	time2, _ := time.Parse(timeTestLayout, "2020-01-02 07:01:00")
+	time3, _ := time.Parse(timeTestLayout, "2020-01-02 07:02:00")
+
+	return []time.Time{time1, time2, time3}
+}
```

Comment:
Try to avoid static data.  Substitute the necessary values inside testCases definitions is more preferable. Just create helper functions. Example below.

Reply:
ok sure. thank you